### PR TITLE
Update software.md

### PR DESCRIPTION
**Pull request by czender**

### CODE REVIEW

**Comment by erget** [comment]
Is this ready for review?

**Comment by czender** [comment]
That's a question for @ethanrd. My contributions are complete and ready for review. BTW, I think the failed Markdown links check can be fixed by upgrading to Python 3.7+. At least that worked to fix the same error message that NCO was getting.

**Comment by ethanrd** [comment]
I added the `cf-view` text above (with minor change) to this PR (mainly just to see if it worked). So I think this PR can be merged once someone checks that my changes don't break anything.

I created another branch and PR with a few other additions and added it to issue #241. More details there, soon.

File: software.md [hunk 1]
```diff
@@ -91,17 +89,15 @@ which uses CF and other metadata convention in its Coordinate System and Scienti
...
 
 ### xarray - Python Package for working with n-D labeled arrays
-[Xarray](http://xarray.pydata.org/) is a python package designed for working with
-labelled, multi-dimensional array data, built around the netCDF data model.
-It uses CF conventions in several ways, such as encoding / decoding variables
-and interpreting metadata for visualization.
+[Xarray](http://xarray.pydata.org/) is a Python package designed for working with labelled, multi-dimensional array data, built around the netCDF data model.
+It uses CF conventions in several ways, such as encoding / decoding variables and interpreting metadata for visualization.
 
 ## Commercial Software Packages
```
Comment (erget) [inline software.md:103]:
```suggestion
## Commercial Software Packages
```

This has no content so I propose removing it.

File: software.md [hunk 2]
```diff
@@ -5,82 +5,80 @@ title: Software that "Understands" CF Data
 
 # Software that "Understands" CF Data
 This page lists sofware packages that "understand" CF Data.
-If you have any additions or corrections for this page,
-please submit an issue on the [CF Website GitHub repo][website-repo]
+If you have any additions or corrections for this page, please submit an issue on the
+[CF Website GitHub repo][website-repo]
 (see the [Contributing Guide][website-contrib] for more details).
 
 For a software package to be listed on this page, it must ....
```
Comment (erget) [inline software.md:12]: Incomplete - should we fill this sentence or discard it?

**Review by erget — COMMENTED** [review]
Had a quick look - looks mostly fine for me but there are 2 things that look funny, what do you think?

File: software.md (hunk 1 above)
Comment (ethanrd) [reply software.md:103]: I'll remove for now. But will ping some Esri folks that have been involved in CF Workshops and our AGU sessions to get some text to add for ArcGIS.

File: software.md (hunk 2 above)
Comment (ethanrd) [reply software.md:12]: I'll discard. We can add back later if we decide we want more detail.

New push to this branch coming shortly.

File: software.md (hunk 1 above)
Comment (ethanrd) [reply software.md]: Removed, will resolve here.

File: software.md (hunk 2 above)
Comment (ethanrd) [reply software.md]: Removed in latest PR. Will mark resolved here.

**Comment by ethanrd** [comment]
I think this one is ready to merge. Can someone take a look and merge if it looks OK?

**Comment by ethanrd** [comment]
Thanks @cofinoa !

**Pull request merged**